### ISSUE TITLE
Upload to community banner photo

### ISSUE THREAD

**Issue by juramos-2020** [cambiatus/frontend#721]
**What we have**
Add a new community configuration option

**Proposal**
Add in community settings an Upload for community banner photo

**Why**
Now communities have a community profile page, which will allow them to have a banner, but we still don't have the option to upload this image in settings

**How**
Add in community settings an Upload for community banner photo
https://www.figma.com/file/nEPG6Dkns8hDVCudVUTEzc/Dashboard?node-id=3434%3A10353

**Comment by NeoVier** [comment]
@juramos-2020 the banner (we called it cover when we implemented it, but I believe it's the same thing) can be uploaded on the "Community Information" page in settings (`/community/settings/info`):

<img width="1288" alt="image" src="https://user-images.githubusercontent.com/39280468/162500858-fe46de6b-1fe2-498f-86ff-6b97750259c4.png">

It's the same image we use in the "Join" page. I think me and @rafachadud even talked about using the same image on both of these scenarios

Not sure if you meant for this issue to review how we present this, but just wanted to make sure you knew we already had this option 😅

**Comment by juramos-2020** [comment]
OI @NeoVier 

sim, estou sabendo que atualmente temos esse campo de upload da imagem em configurações, que usa a mesma imagem tanto para a página de "join" como para o banner de perfil da comunidade.

Mas em alguma reunião - não me recordo qual 😅 - falamos de ter uma opção em configurações para que a comunidade possa fazer esse upload de forma separada.

Será que entendi errado? 😅

**Comment by NeoVier** [comment]
Bom, pode ser... Eu não me recordo, talvez eu não estava junto 😅. Talvez o @lucca65 ou a @muguika possam saber melhor

**Comment by juramos-2020** [comment]
@rafachadud @lucca65 vocês se recordam dessa conversa?

**Comment by lucca65** [comment]
@juramos-2020 isso está implementado no PR do how to earn que está aberto...

podemos fechar junto, oq acham? @juramos-2020 @NeoVier

**Comment by NeoVier** [comment]
@lucca65 No PR do how to earn não mexemos no input, só começamos a mostrar na página sobre a comunidade

**Comment by lucca65** [comment]
Bem, conversamos  nesses dias sobre a foto de banner do perfil da comunidade ser diferente da usada no login, até pq o contexto diferente, daria pra ter duas. Acho que isso é legal, principalmente se tivermos um "preview" simplificado de como fica essa imagem em cada um dos cenários.

Por exemplo: 

<img width="231" alt="Captura de Tela 2022-04-08 às 16 25 37" src="https://user-images.githubusercontent.com/1082127/162510743-9169f42f-14dc-4243-b26d-4ef0b947cc0c.png">

Aí mostramos como ficaria nos dois, trocando a imagem do meio do "mini layout". 

O que acham? Talvez vale a pena até transformar em outra issue, pq essa aqui, como está descrita, está feita, não?

**Comment by juramos-2020** [comment]
OI @lucca65 , sim, para essa issue acho que já temos uma resolução.
Podemos abrir uma nova para mais essa melhoria de "preview".

**Comment by NeoVier** [comment]
Beleza pessoal, vou abrir uma nova issue pra tratarmos do preview.

Até lá, seguimos usamos a mesma imagem em todos os lugares, como já estamos fazendo? Acho que sem o preview ficaria bastante confuso qual imagem serve para que, o que acham?

**Comment by NeoVier** [comment]
Nova issue criada ☝️

**Comment by juramos-2020** [comment]
Oi Henrique

fiz uma descrição na imagem falando para aonde iria cada upload. 
Mas se você acha que ainda tá confuso por não ter o preview e quiser aguardar, tudo bem. Só não tenho certeza se vamos conseguir priorizar isso agora. Mas posso te confirmar depois da nossa reunião de design na segunda.

Eu tinha até feito uma opção usando um tooltip que abria uma ilustração mostrando de qual cenário se tratava. Mas fiquei na dúvida se seria um uso adequado do tooltip 😅
Também pensei em adicionar apenas um link, algo simples, apenas "preview" e abria a ilustração.

**Comment by NeoVier** [comment]
Ainda fico um pouco dividido... Pra gente essa explicação faz sentido, mas imaginando um admin entrando na tela, talvez não faça tanto sentido sem esse contexto. Por outro lado, sei que temos várias outras coisas com maior prioridade agora, e pode demorar um pouco até implementarmos o preview.

Acho que um tooltip não funcionaria muito bem aqui, talvez ficasse melhor se colocássemos na própria página, mas pra fazer isso talvez teríamos que dar uma revisada nessa página de configurações primeiro.

Bom, ainda não tenho muita certeza em como opinar... talvez o @lucca65 ou a @rafachadud tenham outras ideias.

**Comment by NeoVier** [comment]
@lucca65 no lado técnico, hoje a comunidade só tem uma lista de imagens. Continuaríamos com essa lista, e cada índice representa uma imagem diferente (por exemplo, a primeira imagem é a imagem de capa, e a segunda é a do perfil), ou teríamos mudanças na api?

**Comment by juramos-2020** [comment]
Também podemos manter, por enquanto, a mesma imagem para os dois cenários até conseguirmos chegar no preview correto 😉👍

**Comment by rafachadud** [comment]
@NeoVier @juramos-2020 concordo em manter a mesma imagem nos dois cenários. Precisamos repensar as configuracaoes para acomodar melhor a página da comunidade. Talvez algo como o Facebook faz com as páginas faca sentido. Vou explorar essa ideia no perfil do usuário também, porque nao curto muito a imagem de capa estar escondida no formulário. hehe

**Comment by lucca65** [comment]
> nao curto muito a imagem de capa estar escondida no formulário.

+1

**Comment by lucca65** [comment]
@juramos-2020 @rafachadud Muito obrigado vamos implementar
